### ISSUE TITLE
Any interest in using requests-unixsocket?

### ISSUE THREAD

**Issue by msabramo** [docker/docker-py#543]
I extracted code from `docker-py` to create `requests-unixsocket` so that I could use it for other stuff; notably httpie.

I wonder if you want to maybe use requests-unixsocket so that it can benefit from testing and bug fixes from a wider audience?

https://github.com/msabramo/requests-unixsocket

**Comment by TomasTomecek** [comment]
Have you tried putting it directly to requests?

**Comment by msabramo** [comment]
Yes. They were not interested in having it in requests because Unix sockets are not a thing in the HTTP standards. 

See https://github.com/kennethreitz/requests/pull/2355

**Comment by TomasTomecek** [comment]
Right. Didn't know that requests has such philosophy. Thanks for sharing the PR.

Good job on making it your own upstream project!

I think it would be good idea to use your project (but you gotta wait for more official comment from someone from Docker).

**Comment by shin-** [comment]
I'll look into it, might not happen right away since my efforts will go to 1.6 support first. Thanks!
